<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tela_Principal</class>
 <widget class="QMainWindow" name="Tela_Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1318</width>
    <height>859</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Jogue Agora</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>1300</width>
        <height>800</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(218, 218, 218);
background-color: rgb(60, 60, 60);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="5" column="1" rowspan="2">
        <widget class="QFrame" name="frame_11">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="2" rowspan="2">
        <widget class="QFrame" name="frame_16">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QFrame" name="frame_9">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item row="6" column="2" rowspan="2">
        <widget class="QFrame" name="frame_17">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QFrame" name="frame_2">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QFrame" name="frame_20">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QLabel" name="Logo">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>20</y>
               <width>47</width>
               <height>13</height>
              </rect>
             </property>
             <property name="text">
              <string>LOGO</string>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_21">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_22">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    font: 12pt &quot;MS Shell Dlg 2&quot;;
    color: rgb(218, 218, 218);
    background-color: rgb(60, 60, 60);
    border-style: outset;
    border-width: 2px;
    border-color: beige;
    border-radius: 5px;
    padding: 6px;
}

QPushButton:hover {
    background-color: rgb(80, 80, 80); /* Cor de fundo quando o botão é destacado */
}

QPushButton:pressed {
    background-color: rgb(40, 40, 40); /* Cor de fundo quando o botão é pressionado */
}
</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_11">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>SALDO</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="Saldo">
               <property name="styleSheet">
                <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QFrame" name="frame_10">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QFrame" name="frame_19">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QFrame" name="frame_12">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QFrame" name="frame_18">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QFrame" name="frame_8">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2" rowspan="2">
        <widget class="QFrame" name="frame_15">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QFrame" name="frame_23">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QFrame" name="frame_25">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QPushButton" name="Ajuda">
                  <property name="minimumSize">
                   <size>
                    <width>40</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton {
    font: 15pt &quot;MS Shell Dlg 2&quot;;
    color: rgb(218, 218, 218);
    background-color: rgb(50, 50, 50);
    border-style: outset;
    border-width: 0px;
    border-color: beige;
    border-radius: 5px;
    padding: 6px;
}

QPushButton:hover {
    background-color: rgb(88, 88, 88); /* Cor de fundo quando o botão é destacado */
}

QPushButton:pressed {
    background-color: rgb(40, 40, 40); /* Cor de fundo quando o botão é pressionado */
}
</string>
                  </property>
                  <property name="text">
                   <string>Ajuda</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_26">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QPushButton" name="Detalhes">
                  <property name="minimumSize">
                   <size>
                    <width>40</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton {
    font: 15pt &quot;MS Shell Dlg 2&quot;;
    color: rgb(218, 218, 218);
    background-color: rgb(50, 50, 50);
    border-style: outset;
    border-width: 0px;
    border-color: beige;
    border-radius: 5px;
    padding: 6px;
}

QPushButton:hover {
    background-color: rgb(88, 88, 88); /* Cor de fundo quando o botão é destacado */
}

QPushButton:pressed {
    background-color: rgb(40, 40, 40); /* Cor de fundo quando o botão é pressionado */
}
</string>
                  </property>
                  <property name="text">
                   <string>Detalhes</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_24">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QPushButton" name="Perfil_usuario">
                  <property name="minimumSize">
                   <size>
                    <width>40</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton {
    font: 15pt &quot;MS Shell Dlg 2&quot;;
    color: rgb(218, 218, 218);
    background-color: rgb(50, 50, 50);
    border-style: outset;
    border-width: 0px;
    border-color: beige;
    border-radius: 5px;
    padding: 6px;
}

QPushButton:hover {
    background-color: rgb(88, 88, 88); /* Cor de fundo quando o botão é destacado */
}

QPushButton:pressed {
    background-color: rgb(40, 40, 40); /* Cor de fundo quando o botão é pressionado */
}
</string>
                  </property>
                  <property name="text">
                   <string>Perfil</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_27">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QPushButton" name="Sair">
                  <property name="minimumSize">
                   <size>
                    <width>40</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton {
    font: 15pt &quot;MS Shell Dlg 2&quot;;
    color: rgb(218, 218, 218);
    background-color: rgb(50, 50, 50);
    border-style: outset;
    border-width: 0px;
    border-color: beige;
    border-radius: 5px;
    padding: 6px;
}

QPushButton:hover {
    background-color: rgb(88, 88, 88); /* Cor de fundo quando o botão é destacado */
}

QPushButton:pressed {
    background-color: rgb(40, 40, 40); /* Cor de fundo quando o botão é pressionado */
}
</string>
                  </property>
                  <property name="text">
                   <string>Sair</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QFrame" name="frame_13">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="2">
        <widget class="QFrame" name="frame_14">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QFrame" name="frame_28">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QFrame" name="frame_29">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <widget class="QPushButton" name="Redireciona_loja">
                  <property name="minimumSize">
                   <size>
                    <width>40</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton {
    font: 15pt &quot;MS Shell Dlg 2&quot;;
    color: rgb(218, 218, 218);
    background-color: rgb(50, 50, 50);
    border-style: outset;
    border-width: 0px;
    border-color: beige;
    border-radius: 5px;
    padding: 6px;
}

QPushButton:hover {
    background-color: rgb(88, 88, 88); /* Cor de fundo quando o botão é destacado */
}

QPushButton:pressed {
    background-color: rgb(40, 40, 40); /* Cor de fundo quando o botão é pressionado */
}
</string>
                  </property>
                  <property name="text">
                   <string>Loja</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_30">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <widget class="QPushButton" name="extra">
                  <property name="minimumSize">
                   <size>
                    <width>40</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton {
    font: 15pt &quot;MS Shell Dlg 2&quot;;
    color: rgb(218, 218, 218);
    background-color: rgb(50, 50, 50);
    border-style: outset;
    border-width: 0px;
    border-color: beige;
    border-radius: 5px;
    padding: 6px;
}

QPushButton:hover {
    background-color: rgb(88, 88, 88); /* Cor de fundo quando o botão é destacado */
}

QPushButton:pressed {
    background-color: rgb(40, 40, 40); /* Cor de fundo quando o botão é pressionado */
}
</string>
                  </property>
                  <property name="text">
                   <string>Extra</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_31">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <item>
                 <widget class="QPushButton" name="Abr_lista_amigos">
                  <property name="minimumSize">
                   <size>
                    <width>40</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton {
    font: 15pt &quot;MS Shell Dlg 2&quot;;
    color: rgb(218, 218, 218);
    background-color: rgb(50, 50, 50);
    border-style: outset;
    border-width: 0px;
    border-color: beige;
    border-radius: 5px;
    padding: 6px;
}

QPushButton:hover {
    background-color: rgb(88, 88, 88); /* Cor de fundo quando o botão é destacado */
}

QPushButton:pressed {
    background-color: rgb(40, 40, 40); /* Cor de fundo quando o botão é pressionado */
}
</string>
                  </property>
                  <property name="text">
                   <string>Amigos</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_32">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_9">
                <item>
                 <widget class="QPushButton" name="AdicionarFundos">
                  <property name="minimumSize">
                   <size>
                    <width>40</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton {
    font: 15pt &quot;MS Shell Dlg 2&quot;;
    color: rgb(218, 218, 218);
    background-color: rgb(50, 50, 50);
    border-style: outset;
    border-width: 0px;
    border-color: beige;
    border-radius: 5px;
    padding: 6px;
}

QPushButton:hover {
    background-color: rgb(88, 88, 88); /* Cor de fundo quando o botão é destacado */
}

QPushButton:pressed {
    background-color: rgb(40, 40, 40); /* Cor de fundo quando o botão é pressionado */
}
</string>
                  </property>
                  <property name="text">
                   <string>Adicionar Fundos</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" rowspan="8">
        <widget class="QFrame" name="frame_3">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    font: 12pt &quot;MS Shell Dlg 2&quot;;
    color: rgb(218, 218, 218);
    background-color: rgb(60, 50, 70);
    border-style: outset;
    border-width: 0px;
    border-color: beige;
    border-radius: 5px;
    padding: 6px;
}

QPushButton:hover {
    background-color: rgb(80, 80, 80); /* Cor de fundo quando o botão é destacado */
}

QPushButton:pressed {
    background-color: rgb(40, 40, 40); /* Cor de fundo quando o botão é pressionado */
}
</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <widget class="QTableWidget" name="tableWidget">
            <property name="maximumSize">
             <size>
              <width>600</width>
              <height>900</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(120, 120, 120);
background-color: rgb(84, 84, 84);
background-color: rgb(47, 47, 47);</string>
            </property>
            <column>
             <property name="text">
              <string>Nome</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Preco</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>Redireciona_loja</tabstop>
  <tabstop>extra</tabstop>
  <tabstop>Abr_lista_amigos</tabstop>
  <tabstop>AdicionarFundos</tabstop>
  <tabstop>Ajuda</tabstop>
  <tabstop>Detalhes</tabstop>
  <tabstop>Perfil_usuario</tabstop>
  <tabstop>Sair</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
